pkgs/top-level/all-packages.nix

```diff
@@ -32191,7 +32191,7 @@ with pkgs;
 
   tibia = pkgsi686Linux.callPackage ../games/tibia { };
 
-  tintin = callPackage ../games/tintin { };
+  tintin = throw "tintin has been removed due to lack of maintainers";
```

Comment:
Maybe `pkgs/top-level/aliases.nix` is a better choice for this line?

Reply:
Apologies – I didn't see this for some reason before I merged. Should I put in another PR to fix this?

Reply:
> Should I put in another PR to fix this?

That will be nice!